test/hotspot/jtreg/gc/epsilon/TestClasses.java

```diff
@@ -26,12 +26,12 @@
 /**
  * @test TestClasses
  * @requires vm.gc.Epsilon
- * @summary Epsilon is able to allocate a lot of classes
+ * @summary Epsilon is able to allocate a lot of classes, resizing Metaspace
  *
  * @modules java.base/jdk.internal.org.objectweb.asm
  *          java.base/jdk.internal.misc
  *
- * @run main/othervm -Xmx128m -XX:MetaspaceSize=1m -XX:MaxMetaspaceSize=64m -XX:+UnlockExperimentalVMOptions -XX:+UseEpsilonGC -Xlog:gc -Xlog:gc+metaspace gc.epsilon.TestClasses
+ * @run main/othervm -Xmx1g -XX:MetaspaceSize=1m -XX:MaxMetaspaceSize=64m -XX:+UnlockExperimentalVMOptions -XX:+UseEpsilonGC -Xlog:gc -Xlog:gc+metaspace gc.epsilon.TestClasses
```

Comment:
Because this is Epsilon: the new allocations we in field blocks will never be collected. (This is also why the test precomputes the field names, trying not to do that for every class). Look at final "Heap:" messages before/after. Wanted to be on a safe side, so increased it above the 256m it seems to need on my machine. Most other Epsilon tests use `-Xmx1g` anyway.

Reply:
Ah, right. Thanks for the clarification.